api/catalog/settings.py

```diff
@@ -309,6 +309,10 @@
 # The scheme to use for the hyperlinks in the API responses
 API_LINK_SCHEME = config("API_LINK_SCHEME", default="https")
 
+# Trusted origins for CSRF
+# https://docs.djangoproject.com/en/4.0/releases/4.0/#csrf-trusted-origins-changes-4-0
+CSRF_TRUSTED_ORIGINS = ["https://*.openverse.engineering"]
```

Comment:
```suggestion
CSRF_TRUSTED_ORIGINS = config("CSRF_TRUSTED_ORIGINS", default="https://*.openverse.engineering")
```

Would this route be more future-proof if we ever change domains or anything? I think it's fine as is however.

Reply:
Great point! I can update this

Reply:
Update: We have a bunch of other references to `.openverse.engineering` in the `settings.py` file, and since this value is expected to be a list we'll have to do some [unfortunate munging](https://pypi.org/project/python-decouple/#understanding-the-cast-argument) if we want it to be configurable with an environment variable. So I'm going to leave this as-is for now :slightly_smiling_face: